web/pdf_find_controller.js

```diff
@@ -341,10 +341,10 @@ class PDFFindController {
       this._pdfViewer.currentPageNumber = index + 1;
     }
 
-    const page = this._pdfViewer.getPageView(index);
-    if (page.textLayer) {
-      page.textLayer.updateMatches();
-    }
+    this._eventBus.dispatch('updatematches', {
```

Comment:
It might not hurt to use a slightly more specific name here, so how about e.g. `updatetextlayermatches` instead.

Reply:
Done in the new commit series.